3.3-nginx/Dockerfile

```diff
@@ -28,7 +27,8 @@ RUN apt-get update \
  && sed -i /etc/modsecurity.d/modsecurity.conf \
       -e 's/SecRuleEngine DetectionOnly/SecRuleEngine On/g'
 
-COPY docker-entrypoint.sh /
+COPY 3.3-nginx/docker-entrypoint.sh /
+COPY common/modsec-activate-rules.sh /
 
 EXPOSE 80
```

Comment:
```suggestion
EXPOSE ${PORT}
```

Reply:
Actually I left the configuration for nginx for the next pull request that I'm currently working on. I had to touch the nginx part of the repository because of the restructuring of the project that was suggested by @bittner. For that matter I created a new commit on top of the "Add environment variables" in this pull request.